### PR TITLE
Add GHRSST MetOp-B SST for obs prep

### LINKED ISSUE #609: GHRSST (AVHRR, VIIRS, does that also include the MW instruments?)

### PR DESCRIPTION
**Pull request by ShastriPaturi**
Partially resolves #609

### CODE REVIEW

File: utils/test/prepdata.sh [hunk 1]
```diff
@@ -4,7 +4,5 @@ set -e
 project_source_dir=$1
 
 ncgen -o rads_adt_j3_2021182.nc4 ${project_source_dir}/testdata/rads_adt_j3_2021182.cdl
-ncgen -o icec_amsr2_north_1.nc4 ${project_source_dir}/testdata/icec_amsr2_north_1.cdl
-ncgen -o icec_amsr2_north_2.nc4 ${project_source_dir}/testdata/icec_amsr2_north_2.cdl
-ncgen -o icec_amsr2_south_1.nc4 ${project_source_dir}/testdata/icec_amsr2_south_1.cdl
-ncgen -o icec_amsr2_south_2.nc4 ${project_source_dir}/testdata/icec_amsr2_south_2.cdl
+ncgen -o ghrsst_sst_mb_202107010000.cdl ${project_source_dir}/testdata/ghrsst_sst_mb_202107010000.cdl
+ncgen -o ghrsst_sst_mb_202107010100.cdl ${project_source_dir}/testdata/ghrsst_sst_mb_202107010100.cdl
```
Comment (guillaumevernieres) [inline utils/test/prepdata.sh]: why did you remove the amsr2 files @ShastriPaturi ?

File: utils/test/prepdata.sh [hunk 2]
```diff
@@ -4,7 +4,5 @@ set -e
 project_source_dir=$1
 
 ncgen -o rads_adt_j3_2021182.nc4 ${project_source_dir}/testdata/rads_adt_j3_2021182.cdl
-ncgen -o icec_amsr2_north_1.nc4 ${project_source_dir}/testdata/icec_amsr2_north_1.cdl
-ncgen -o icec_amsr2_north_2.nc4 ${project_source_dir}/testdata/icec_amsr2_north_2.cdl
-ncgen -o icec_amsr2_south_1.nc4 ${project_source_dir}/testdata/icec_amsr2_south_1.cdl
-ncgen -o icec_amsr2_south_2.nc4 ${project_source_dir}/testdata/icec_amsr2_south_2.cdl
```
Comment (AndrewEichmann-NOAA) [inline utils/test/prepdata.sh:10]: I thought you were ok with these!

**Review by guillaumevernieres — CHANGES_REQUESTED** [review]

File: utils/test/prepdata.sh (hunk 2 above)
Comment (ShastriPaturi) [reply utils/test/prepdata.sh:10]: @AndrewEichmann-NOAA I was...
While merging conflict... I deleted them,

Comment (ShastriPaturi) [reply utils/test/prepdata.sh:10]: Let me add them back

**Review by AndrewEichmann-NOAA — APPROVED** [review]
thumbs up

**Review by guillaumevernieres — APPROVED** [review]
